pkg/resource/scalable_target/custom_api.go

```diff
@@ -35,8 +35,15 @@ func (rm *resourceManager) customDescribeScalableTarget(
 	}
 }
 
+// customSetOutputUpdate sets the LastModifiedTime field to the creationTime
```

Comment:
nit: method name not matching in comment, same for other

and in future lets just rename this file to hooks.go

Reply:
fixed the comments, we can change the filename in another PR.